### PR TITLE
Make shellbase buildable on Scala 2.12

### PR DESCRIPTION
**Pull request by zhalas**

### CODE REVIEW

**Comment by cddude229** [comment]
```
lang/scala-library/2.11.6/scala-library-2.11.6.pom (1.4 kB at 285 kB/s)
[ERROR] error: scala.reflect.internal.MissingRequirementError: object java.lang.Object in compiler mirror not found.
[ERROR] 	at scala.reflect.internal.MissingRequirementError$.signal(MissingRequirementError.scala:17)
[ERROR] 	at scala.reflect.internal.MissingRequirementError$.notFound(MissingRequirementError.scala:18)
```

Seems legit?

**Comment by zhalas** [comment]
@cddude229 at first glance looks like some build infra issue. Probably worth retriggering before investigating further.

**Comment by fdaca** [comment]
The build issue should be fixed by this change: https://github.com/SumoLogic/shellbase/pull/40

**Comment by fdaca** [comment]
@zhalas Please, try to rebase your banch and merge into SumoLogic:master

File: shellbase-slack/pom.xml [hunk 1]
```diff
@@ -31,16 +31,16 @@
 
     <dependency>
       <groupId>com.flyberrycapital</groupId>
-      <artifactId>scala-slack_2.11</artifactId>
-      <version>0.2.0</version>
+      <artifactId>scala-slack_${scala.version.major}</artifactId>
```
Comment (cddude229) [inline shellbase-slack/pom.xml]: For the record: bumping this is considered an API change, so we'll need to bump minor version on release.

**Pull request merged**